app/assets/javascripts/3dbio_viewer/src/webapp/components/HtmlTooltip.tsx

```diff
@@ -0,0 +1,40 @@
+import { Theme, Typography, withStyles } from "@material-ui/core";
+import { Tooltip } from "@material-ui/core";
+import styled from "styled-components";
+
+export const HtmlTooltip = withStyles((theme: Theme) => ({
```

Comment:
[nothing do do] HOC-styling (Higher Order Components, in general) has gone out of fashion, but I see this is coming from the covid19 app, so let's just leave it.